bot/exts/fun/uwu.py

```diff
@@ -178,19 +181,28 @@ async def uwu_command(self, ctx: Context, *, text: t.Optional[str] = None) -> No
         await clean_content(fix_channel_mentions=True).convert(ctx, text)
 
         fun_cog: t.Optional[Fun] = ctx.bot.get_cog("Fun")
+
         if fun_cog:
             text, embed = await fun_cog._get_text_and_embed(ctx, text)
 
-            # Grabs the text from the embed for uwuification.
+            # Grabs the text from the embed for uwuification
             if embed is not None:
                 embed = fun_cog._convert_embed(self._uwuify, embed)
+            else:
+                # Only use the first embed since only a single one can be sent
+                embed = fun_cog._convert_embed(self._uwuify, embeds[0]) if embeds else None
         else:
             embed = None
+
         converted_text = self._uwuify(text)
         converted_text = helpers.suppress_links(converted_text)
 
         # Adds the text harvested from an embed to be put into another quote block.
-        converted_text = f">>> {converted_text.lstrip('> ')}"
+        if text:
+            converted_text = f">>> {converted_text.lstrip('> ')}"
```

Comment:
I would consider moving lines 197 198:
```py
converted_text = self._uwuify(text)
converted_text = helpers.suppress_links(converted_text)
```
to be within the `if text:` block as well, no point running those if we aren't going to use the result.